src/createAdminAccountScript.ts

```diff
@@ -1,88 +1,114 @@
-// Step by step
-// 1. Connect DB
-console.log('Admin entered: ' + process.argv[2]);
-
 // Import functions
 
 import generator from 'generate-password';
-
-
 import nodemailer from 'nodemailer';
+
 import User from './models/User';
 import connectDatabase from './database';
 import { hashPassword } from './helpers/passwordHash';
+import { createJWT } from './helpers/jwtHelper';
+import Token from './models/Token';
+import { exit } from 'node:process';
+
 require('dotenv').config();
 
 const mailgun = require('mailgun-js')({
   apiKey: process.env.API_KEY,
   domain: process.env.DOMAIN
 });
+const getUserByEmail = async (email: string) => {
+  await connectDatabase();
```

Comment:
It seems that we will `connectDatabase` anyway, so how about we connect to database somewhere earlier in the code so that `getUserByEmail`, `getTokenByUserId` and `deleteTokenByUserId` does not have to call `connectDatabase`?